handlers/analytics/log_processor.go

```diff
@@ -58,7 +58,9 @@ type LogData struct {
 	PlaybackID            string       `json:"playback_id"`
 	ViewerHash            string       `json:"viewer_hash"`
 	Protocol              string       `json:"protocol"`
-	PageURL               string       `json:"page_url"`
+	Domain                string       `json:"domain"`
+	Path                  string       `json:"path"`
+	Params                string       `json:"params"`
```

Comment:
I'd like to see `Hash` here as well - it likely is the least important or interesting value in a URL, but some apps use it for routing and it would be a field we shouldn't drop: https://developer.mozilla.org/en-US/docs/Web/API/URL/hash

Reply:
Added